Source/Private/TextExperiment/Utility/NSParagraphStyle+ASText.m

```diff
@@ -25,13 +25,15 @@ + (NSParagraphStyle *)as_styleWithCTStyle:(CTParagraphStyleRef)CTStyle {
   
   NSMutableParagraphStyle *style = [[NSParagraphStyle defaultParagraphStyle] mutableCopy];
   
+#if TARGET_OS_IOS
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
   CGFloat lineSpacing;
   if (CTParagraphStyleGetValueForSpecifier(CTStyle, kCTParagraphStyleSpecifierLineSpacing, sizeof(CGFloat), &lineSpacing)) {
```

Comment:
Need to add back support for line spacing on tvOS since `kCTParagraphStyleSpecifierLineSpacing` is unavailable on tvOS.